packages/liveblocks-client/src/LiveObject.ts

```diff
@@ -55,14 +56,20 @@ export class LiveObject<O extends LsonObject> extends AbstractCrdt {
   /**
    * @internal
    */
-  _serialize(parentId?: string, parentKey?: string, doc?: Doc): Op[] {
+  _serialize(parentId: string, parentKey: string, doc?: Doc): CreateChildOp[];
+  _serialize(
+    parentId?: undefined,
+    parentKey?: undefined,
+    doc?: Doc
+  ): CreateOp[];
+  _serialize(parentId?: string, parentKey?: string, doc?: Doc): CreateOp[] {
```

Comment:
Overloaded here, only for `LiveObject`. When `parentId` and `parentKey` aren't provided, this will return a `CreateOp[]`, otherwise, all the entries in here will be `CreateChildOp[]` (which excludes the possibility of having a "root object" op in there).